src/lhttpc_lb.erl

```diff
@@ -21,6 +24,7 @@
                 free=[] :: list()}).
 
 -export_types([host/0, port/0, max_connections/0, connection_timeout/0]).
+-type proplist() :: [{atom(),any()}].
```

Comment:
tuplelist() or kvlist() also work fine. tagged_tuples() is good too, as it gets rid of the confusion.